client/ProfileComponents/Profile.jsx

```diff
@@ -17,21 +16,21 @@ class Profile extends React.Component {
   render() {
     return (
       <div>
-        <div className="pic-position">
-          <Pic photo={this.props.fakePhoto} />
-        </div>
-        <div className="bio-position">
-          <BioList userInfo={this.props.fakeUser} />
-        </div>
-        <div className="posts-position">
-          <PostList posts={this.props.fakePosts} />
-        </div>
-        <div className="friends-position">
-          <Friends friends={this.props.fakeFriends} />
-        </div>
+        <h1>
+          <small>Profile Page for HR Peeps</small>

```

Comment:
Reconsider the `<small>` tag?  Maybe replace with `<h5>`?  Alternatively, we can just put it in a `<p><span></span></p>` and style the font-size of the `<span>` tag to be very small.